sdk/src/actor.rs

```diff
@@ -72,25 +70,28 @@ pub fn install_actor(code_cid: &Cid) -> SyscallResult<()> {
 
 /// Determines whether the supplied CodeCID belongs to a built-in actor type,
 /// and to which.
-pub fn get_builtin_actor_type(code_cid: &Cid) -> Option<actor::builtin::Type> {
+pub fn get_builtin_actor_type(code_cid: &Cid) -> Option<i32> {
```

Comment:
i32?

Reply:
yes, after a certain point i32 were being converted into the type anyways.

Reply:
u32 would be better, but I'm trying to avoid breaking the API too much. Once we've backported this to v2, we can change v3.